integration/client/go.mod

```diff
@@ -15,9 +15,26 @@ require (
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.8.0
+	github.com/sirupsen/logrus v1.7.0
```

Comment:
I updated the `go.mod` by removing all `require` rules, except for github.com/Microsoft/hcsshim/test, and let `go mod tidy` add back the versions (based on the go mod dependency tree)